pkg/apis/kubermatic/v1/datacenter.go

```diff
@@ -249,7 +267,7 @@ type DatacenterSpec struct {
 
 	//nolint:staticcheck
 	//lint:ignore SA5008 omitgenyaml is used by the example-yaml-generator
-	Fake *DatacenterSpecFake `json:"fake,omitempty,omitgenyaml"`
+	Fake *DatacenterSpecFake `json:"fake,omitempty,omitempty"`
```

Comment:
Why did you change this?

Reply:
I didn't do this explicitly, must have been either some script or auto-correct on save.

Reply:
Please undo and re-generate the docs.

Reply:
This can happen if you run a update-docs.sh script and interrupt it, then the magic combination of `sed` commands will leave you in this half-ready half-broken state.

Reply:
Yes, I figured that out. I'm fixing it right now and will fix it in master as well ✌️

Reply:
Done